spark-knn-examples/src/main/scala/com/github/saurfang/spark/ml/knn/examples/MNIST.scala

```diff
@@ -17,10 +17,11 @@ object MNIST {
     import spark.implicits._
 
     //read in raw label and features
-    val dataset = MLUtils.loadLibSVMFile(sc, "data/mnist/mnist.bz2")
+    var dataset = MLUtils.loadLibSVMFile(sc, "data/mnist/mnist.bz2")
```

Comment:
Can you change this back to `val`? Either nest the function call or create another variable like `rawDataset`